### PR TITLE
Issue #2 - Base 'require' on process.cwd instead of __dirname

### LINKED ISSUE #2: [Feature] - Resolve require calls relatively to `process.cwd()` instead of `__dirname`
I would like to require files in the notebook; however I'm not sure how the files are resolved...
I tried to `require` relatively to the notebook and the `jupyter` root directory but both failed.

### PR DESCRIPTION
**Pull request by 3Nigma**
... this way, the path root is set to the notebook's base directory.

Closes #2

### CODE REVIEW

File: src/session/message_loop.js [hunk 1]
```diff
@@ -45,7 +45,18 @@ class MessageLoop {
             clearImmediate: (t) => { if (t instanceof SessionClearableTimer) t._clear(); else clearImmediate(t) }, 
             clearInterval: (t) => { if (t instanceof SessionClearableTimer) t._clear(); else clearInterval(t) }, 
             clearTimeout: (t) => { if (t instanceof SessionClearableTimer) t._clear(); else clearTimeout(t) },
-            exports, module, require,
+            exports, module, 
+            require: (id) => {
```
Comment (3Nigma) [inline src/session/message_loop.js:49]: Node [require](https://nodejs.org/api/modules.html#modules_require_id) docs (for reference)

**Comment by goranmoomin** [comment]
@3Nigma I’m on my phone, so I can’t check, but if my understanding is correct, that won’t allow `require`ing modules installed in `node_modules`... so I think that the `else` branch should be something more like: 
```js
finalRequirePath = path.join(process.cwd(), 'node_modules', id);
```

**Comment by goranmoomin** [comment]
@3Nigma I was forgetting about the [`require.resolve`](https://nodejs.org/dist/latest-v12.x/docs/api/modules.html#modules_require_resolve_request_options) function. One of the options is `path`, which AFAIK allows to resolve modules in an arbitrary path. It would be something more like:
```js
finalRequirePath = require.resolve(id, {
    paths: process.cwd()
});
```

**Comment by 3Nigma** [comment]
@pcr910303 
**Good one!** I like the 
```js
finalRequirePath = require.resolve(id, {
    paths: [ process.cwd() ]
});
```
approach. It's better than trying to manually build the path. Let's use that.

**Pull request merged**